# Will allow to handle move signal outside from mozilla/widget/gtk2
Index: mozilla/widget/src/gtk2/nsWindow.cpp
===================================================================
--- mozilla.orig/widget/src/gtk2/nsWindow.cpp
+++ mozilla/widget/src/gtk2/nsWindow.cpp
@@ -3016,21 +3016,21 @@ nsWindow::NativeCreate(nsIWidget        
                                G_CALLBACK(theme_changed_cb), this);
     }
 
     if (mContainer) {
         g_signal_connect_after(G_OBJECT(mContainer), "size_allocate",
                                G_CALLBACK(size_allocate_cb), NULL);
         g_signal_connect(G_OBJECT(mContainer), "expose_event",
                          G_CALLBACK(expose_event_cb), NULL);
-        g_signal_connect(G_OBJECT(mContainer), "enter_notify_event",
+        g_signal_connect_after(G_OBJECT(mContainer), "enter_notify_event",
                          G_CALLBACK(enter_notify_event_cb), NULL);
-        g_signal_connect(G_OBJECT(mContainer), "leave_notify_event",
+        g_signal_connect_after(G_OBJECT(mContainer), "leave_notify_event",
                          G_CALLBACK(leave_notify_event_cb), NULL);
-        g_signal_connect(G_OBJECT(mContainer), "motion_notify_event",
+        g_signal_connect_after(G_OBJECT(mContainer), "motion_notify_event",
                          G_CALLBACK(motion_notify_event_cb), NULL);
         g_signal_connect(G_OBJECT(mContainer), "button_press_event",
                          G_CALLBACK(button_press_event_cb), NULL);
         g_signal_connect(G_OBJECT(mContainer), "button_release_event",
                          G_CALLBACK(button_release_event_cb), NULL);
         g_signal_connect(G_OBJECT(mContainer), "focus_in_event",
                          G_CALLBACK(focus_in_event_cb), NULL);
         g_signal_connect(G_OBJECT(mContainer), "focus_out_event",
